### ISSUE TITLE
Voting Synchronization

### ISSUE THREAD

**Issue by zazwaz12** [narmirarmi/CITS3403_Group_Project#11]
When the votes are shown, they update on the backend, but only refresh on reload. 

When the user votes, the vote count needs to increment.

It had live refresh before when only one vote would show, now that all the votes show they do not refresh live.

I think this may need to call the database version on the refresh.

**Comment by narmirarmi** [comment]
is this in a specific branch?

**Comment by zazwaz12** [comment]
Yes, but I think its merged with the database branch, as we need to sort out the database functionality as a higher priority.

**Comment by narmirarmi** [comment]
Sort it out in what sense? Also modifications were being made on the origin/database-python branch, so the origin/database branch has been merged to main, and most / all changes to the database going fotrward will be on origin/database-python. We can move to teams since this doesn't directly relate to the issue above!

**Comment by zazwaz12** [comment]
Resolved this. In data-from-database commit **ecbda701fcf6c719ebc1ca3f02fa95114c845fb5**.
